package.json

```
@@ -1,67 +1,56 @@
 {
   "name": "@testing-library/user-event",
-  "version": "0.0.0-development",
-  "description": "Simulate user events for react-testing-library",
-  "keywords": [
-    "react-testing-library",
-    "dom-testing-library",
-    "react",
-    "testing"
-  ],
+  "version": "0.0.0-semantically-released",
+  "description": "Fire events the same way the user does",
   "main": "dist/index.js",
   "typings": "typings/index.d.ts",
-  "scripts": {
-    "test": "jest",
-    "test:coverage": "npm t -- --coverage",
-    "codecov": "codecov",
-    "build": "babel src -d dist",
-    "semantic-release": "semantic-release",
-    "travis-deploy-once": "travis-deploy-once",
-    "cm": "npx git-cz",
-    "contributors:add": "all-contributors add",
-    "contributors:generate": "all-contributors generate"
+  "keywords": [],
```

`keywords` were deleted

```suggestion
  "keywords": [
    "react-testing-library",	
    "dom-testing-library",	
    "react",	
    "testing"	
  ],
```